.github/workflows/codeql-analysis.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       uses: actions/checkout@v3
 
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@v2
+      uses: github/codeql-action/init@v1
```

Comment:
Why can't this remain `v2`?  Please clarify.

Reply:
@jww3 This commit c8466ed95778604cdd7ea5a4875e87574070ed4c  change has been  merged from main to this branch.

Reply:
Hmm this shouldn't have happened, this has to stay `@v2`. In fact this entire file should not be part of this PR. I'll update it

Reply:
@megamanics - I'm not able to push to your fork i.e. this PR. We could do a rebase to fix this but this is simpler:
```
git checkout origin/main -- .github/workflows/codeql-analysis.yml
```
Could you please pull the workflow file from the main branch, commit and push it?